feature-libs/checkout/base/components/checkout-delivery-mode/checkout-delivery-mode.component.html

```diff
@@ -13,16 +13,34 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
   <div [formGroup]="mode" class="cx-delivery-mode-wrapper">
     <div role="radiogroup">
       <div class="form-check" *ngFor="let mode of supportedDeliveryModes">
-        <input
-          [attr.aria-checked]="getAriaChecked(mode.code)"
-          class="form-check-input"
-          role="radio"
-          type="radio"
-          id="deliveryMode-{{ mode.code }}"
-          (change)="changeMode(mode.code)"
-          [value]="mode.code"
-          formControlName="deliveryModeId"
-        />
+        <!-- TODO: (CXSPA-6009) Remove feature flags next major release -->
+        <ng-container *cxFeatureLevel="'6.8'">
```

Comment:
The only change I see here is adding the key event. I wouldn't consider this as "breaking" so I don't think that we need the feature flags.

Reply:
I got rid of feature flags. ;)